html/modules/custom/reliefweb_topics/reliefweb_topics.module

```diff
@@ -0,0 +1,111 @@
+<?php
+
+/**
+ * @file
+ * Code for the ReliefWeb Topics feature.
+ */
+
+/**
+ * Get all community topics.
+ */
+function reliefweb_topics_get_all() {
+  return \Drupal::state()->get('reliefweb_topics', []);
+}
```

Comment:
It's only for the community topics, so I would suggest to rename this function and the state to make it clearer.